### ISSUE TITLE
spotlessJavaCheck fails in main

### ISSUE THREAD

**Issue by daniel-iobuilders** [hyperledger/besu#2642]
### Description
There are `spotless` errors in the submodule `qbft-ref-tests`. Because of those errors the gradle `build` task fails. The pipeline does not show the error, my best guess is that it does no update the submodules before running spotlessJavaCheck.

### Steps to Reproduce (Bug)
1. docker run --rm -it openjdk:11 bash
2. git clone https://github.com/hyperledger/besu.git
3. cd besu/
4. git submodule init
5. git submodule update
6. ./gradlew spotlessJavaCheck

**Expected behavior:** spotlessJavaCheck should not fail

**Actual behavior:** fails with the following error:
```
> The following files had format violations:
      consensus/qbft/src/reference-test/resources/docker_interop/app/src/main/java/tech/pegasys/qbft/interop/DockerComposeService.java
          @@ -19,8 +19,50 @@
           import·java.util.Map;
           
           public·class·DockerComposeService·{
          -····private·static·final·String·BESU_COMMAND·=·"chmod·755·/scripts/run_besu.sh·&&·/scripts/run_besu.sh";
          -····private·static·final·String·GO_QUORUM_COMMAND·=·"chmod·755·/scripts/run_geth.sh·&&·/scripts/run_geth.sh";
          +··private·static·final·String·BESU_COMMAND·=
          +······"chmod·755·/scripts/run_besu.sh·&&·/scripts/run_besu.sh";
          +··private·static·final·String·GO_QUORUM_COMMAND·=
          +······"chmod·755·/scripts/run_geth.sh·&&·/scripts/run_geth.sh";
          +··private·String·serviceName;
          +··private·boolean·isBesuMode;
          +··private·String·hostname;
          +··private·int·rpcPort;
          +··private·int·p2pPort;
          +··private·String·ipAddress;
          +··private·String·privateKey;
          +··private·String·staticNodeFileName;
          +
          +··public·Map<String,·Object>·getMap()·{
          +····final·Map<String,·Object>·map·=·new·LinkedHashMap<>();
          +····map.put("hostname",·hostname);
          +····map.put("image",·isBesuMode·?·"${BESU_IMAGE}"·:·"${QUORUM_IMAGE}");
          +····map.put(
          +········"ports",
          +········List.of(
          +············p2pPort·+·":"·+·p2pPort·+·"/tcp",
          +············p2pPort·+·":"·+·p2pPort·+·"/udp",
          +············rpcPort·+·":"·+·rpcPort));
          +····map.put("entrypoint",·List.of("/bin/sh"));
          +····map.put("command",·List.of("-c",·getCommand()));
          +····map.put("networks",·Map.of("app_net",·Map.of("ipv4_address",·ipAddress)));
          +····map.put("restart",·"no");
          +····map.put("volumes",·List.of("./:/scripts"));
          +
          +····return·Map.of(serviceName,·map);
          +··}
          +
          +··private·String·getCommand()·{
          +····final·String·commandFormat·=·"%s·%s·%d·%d·%s";
          +····return·String.format(
          +········commandFormat,
          +········isBesuMode·?·BESU_COMMAND·:·GO_QUORUM_COMMAND,
          +········privateKey,
          +········p2pPort,
          +········rpcPort,
          +········staticNodeFileName);
          +··}
          +
      ... (145 more lines that didn't fit)
  Violations also present in:
      consensus/qbft/src/reference-test/resources/docker_interop/app/src/main/java/tech/pegasys/qbft/interop/DockerComposeYaml.java
      consensus/qbft/src/reference-test/resources/docker_interop/app/src/main/java/tech/pegasys/qbft/interop/QbftDockerComposeApp.java
  Run './gradlew :spotlessApply' to fix these violations.
```

**Frequency:**  100%

**Comment by vmichalik** [comment]
thanks @daniel-iobuilders! we'll look into this

**Comment by yashar2004** [comment]
I have the same problem

### Steps to Reproduce (Bug)
git clone --recursive https://github.com/hyperledger/besu
cd besu
./gradlew build 

**Expected behavior:** spotlessJavaCheck should not fail

**Actual behavior:** fails with the following error:

`> Task :spotlessJavaCheck FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':spotlessJavaCheck'.
> The following files had format violations:
      consensus/qbft/src/reference-test/resources/docker_interop/app/src/main/java/tech/pegasys/qbft/interop/DockerComposeService.java
          @@ -19,8 +19,50 @@
           import·java.util.Map;
           
           public·class·DockerComposeService·{
          -····private·static·final·String·BESU_COMMAND·=·"chmod·755·/scripts/run_besu.sh·&&·/scripts/run_besu.sh";
          -····private·static·final·String·GO_QUORUM_COMMAND·=·"chmod·755·/scripts/run_geth.sh·&&·/scripts/run_geth.sh";
          +··private·static·final·String·BESU_COMMAND·=
          +······"chmod·755·/scripts/run_besu.sh·&&·/scripts/run_besu.sh";
          +··private·static·final·String·GO_QUORUM_COMMAND·=
          +······"chmod·755·/scripts/run_geth.sh·&&·/scripts/run_geth.sh";
          +··private·String·serviceName;
          +··private·boolean·isBesuMode;
          +··private·String·hostname;
          +··private·int·rpcPort;
          +··private·int·p2pPort;
          +··private·String·ipAddress;
          +··private·String·privateKey;
          +··private·String·staticNodeFileName;
          +
          +··public·Map<String,·Object>·getMap()·{
          +····final·Map<String,·Object>·map·=·new·LinkedHashMap<>();
          +····map.put("hostname",·hostname);
          +····map.put("image",·isBesuMode·?·"${BESU_IMAGE}"·:·"${QUORUM_IMAGE}");
          +····map.put(
          +········"ports",
          +········List.of(
          +············p2pPort·+·":"·+·p2pPort·+·"/tcp",
          +············p2pPort·+·":"·+·p2pPort·+·"/udp",
          +············rpcPort·+·":"·+·rpcPort));
          +····map.put("entrypoint",·List.of("/bin/sh"));
          +····map.put("command",·List.of("-c",·getCommand()));
          +····map.put("networks",·Map.of("app_net",·Map.of("ipv4_address",·ipAddress)));
          +····map.put("restart",·"no");
          +····map.put("volumes",·List.of("./:/scripts"));
          +
          +····return·Map.of(serviceName,·map);
          +··}
          +
          +··private·String·getCommand()·{
          +····final·String·commandFormat·=·"%s·%s·%d·%d·%s";
          +····return·String.format(
          +········commandFormat,
          +········isBesuMode·?·BESU_COMMAND·:·GO_QUORUM_COMMAND,
          +········privateKey,
          +········p2pPort,
          +········rpcPort,
          +········staticNodeFileName);
          +··}
          +
      ... (145 more lines that didn't fit)
  Violations also present in:
      consensus/qbft/src/reference-test/resources/docker_interop/app/src/main/java/tech/pegasys/qbft/interop/DockerComposeYaml.java
      consensus/qbft/src/reference-test/resources/docker_interop/app/src/main/java/tech/pegasys/qbft/interop/QbftDockerComposeApp.java
  Run './gradlew :spotlessApply' to fix these violations.
`

**Comment by jframe** [comment]
This has been fixed as part of #2761 which excludes the qbft reference tests from the spotless check.

Agree that we should make that QBFT reference tests are using google code format.
